reporter-web-app/src/components/PackageFindingsTable.js

```diff
@@ -110,18 +101,7 @@ class PackageFindingsTable extends React.Component {
                 dataIndex: 'path',
                 defaultSortOrder: 'ascend',
                 key: 'path',
-                sorter: (a, b) => {
-                    const idA = a.path.length;
-                    const idB = b.path.length;
-                    if (idA < idB) {
-                        return -1;
-                    }
-                    if (idA > idB) {
-                        return 1;
-                    }
-
-                    return 0;
-                },
+                sorter: (a, b) => a.path.localeCompare(b.path),
```

Comment:
This now sorts by path name, not by path length anymore.

Reply:
Revert my change in sorting scan results file finding, will try out other options and will fill a new PR for this in the near future